core-to-plc/src/Language/Plutus/CoreToPLC.hs

```diff
@@ -8,8 +8,10 @@
 module Language.Plutus.CoreToPLC where
```

Comment:
Maybe split this file somehow? It's so big.

Reply:
Yeah, I've tried a few times. Unfortunately converting types and terms ends up being mutually recursive. I think I might just bite the bullet and add some `.hs-boot` files to make it work.